dev/com.ibm.ws.logging/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -103,3 +103,9 @@ json.access.log.fields.logFormat=Use the set of access log fields that match log
 
 apps.write.json=Allow apps to write JSON
 apps.write.json.desc=When message log or console log is in JSON format, allow applications to write JSON-formatted messages to those destinations, without modification.
+
+rollover.start.time=Rollover start time for time based log rollover
+rollover.start.time.desc=The scheduled time of day when logs should first rollover. The rollover interval duration will begin at rollover start time. Valid values follow a 24-hour ISO-8601 datetime format of HH:MM, where 00:00 represents midnight. Padding zeroes are required. If rolloverInterval is specified, the default value of rolloverStartTime is set to 00:00, midnight. If both rolloverInterval and rolloverStartTime are unspecified, time based log rollover is disabled.
```

Comment:
- We try to not use "should" but the first sentence is clear (thus, okay to keep "should"). If you keep the "should," change "rollover" to "roll over"

  If you want to remove the "should," maybe use one of the following sentences instead:
   - The scheduled time of day when logs first roll over.
   - The scheduled time of day for the first log rollover.
   - The scheduled time of day when log rollover starts.
   - The scheduled start time for log rollover.

- Change "will begin" to "begins"

- Change "zeroes" to "zeros"